### PR TITLE
Simplize API Reference Documentation

### PR DESCRIPTION
**Pull request by reyoung**
default
default
default
default

Polish `fill_constant_batch_size_like`, `load`, `max_sequence_len`, `resize_bilinear2d`:

* Add missing example
* Move `load` from `tensor` to `io` 
* Use C++ comments instead of copying them twice.

### CODE REVIEW

File: paddle/fluid/operators/batch_size_like.h [hunk 1]
```diff
@@ -54,18 +54,18 @@ class BatchSizeLikeOp : public framework::OperatorWithKernel {
...
-             "(Tensor) Tensor "
-             "whose input_dim_idx'th dimension specifies the batch_size");
+    AddInput(
+        "Input",
+        "Tensor whose input_dim_idx'th dimension specifies the batch_size");
     AddOutput("Out",
-              "(Tensor) Tensor of specified shape will be filled "
+              "Tensor of specified shape will be filled "
               "with the specified value");
-    AddAttr<std::vector<int>>("shape", "(vector<int>) The shape of the output");
```
Comment (reyoung) [inline paddle/fluid/operators/batch_size_like.h]: It is stupid to write type again. The type has been registered by template arguments.

**Comment by skylarch** [comment]
第一个API的indentation，从“It also sets..."开始错位。

**Comment by reyoung** [comment]
@skylarch  Done.

**Comment by JiayiFeng** [comment]
`max_seuqence_len`的文档多了一项“返回类型”

**Comment by reyoung** [comment]
@JiayiFeng  Done

**Comment by reyoung** [comment]
Also polish the math equations in crf layer

default

File: python/paddle/fluid/layers/layer_function_generator.py [hunk 2]
```diff
@@ -238,32 +241,47 @@ def templatedoc():
...
-            comment += line
-            comment += "\n"
-
-        args = {"comment": comment}
+            line = line.strip()
+            if len(line) != 0:
+                comment += escape_inline_math(line)
+                comment += " "
+            elif len(comment) != 0:
+                comment += "\n    \n    "
```
Comment (typhoonzero) [inline python/paddle/fluid/layers/layer_function_generator.py]: Why need to add spaces at then end?

File: python/paddle/fluid/layers/layer_function_generator.py [hunk 3]
```diff
@@ -238,32 +241,47 @@ def templatedoc():
...
+
+        args = {"comment": trim_ending_dot(comment)}
         for each_input in op_proto.inputs:
             input_name = _convert_(each_input.name)
```
Comment (typhoonzero) [inline python/paddle/fluid/layers/layer_function_generator.py]: `_convert_` seems will output wired string when input is something like `ROIs`, outputs `ro_is`. Just put out this problem.

Or maybe we can just check that name is valid?

File: python/paddle/fluid/layers/layer_function_generator.py (hunk 2 above)
Comment (reyoung) [reply python/paddle/fluid/layers/layer_function_generator.py]: It correctly indent the comment to 4 spaces. It needed for google style comments.

It is tricky now. It should use regex to detect indent length.

**Comment by shanyi15** [comment]
@skylarch , please recheck the format

**Pull request merged**